src/ClientData/include/ClientData/FunctionInfo.h

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_FUNCTION_INFO_H_
+#define CLIENT_DATA_FUNCTION_INFO_H_
+
+#include <optional>
+#include <string>
+#include <utility>
+
+#include "GrpcProtos/symbol.pb.h"
+
+namespace orbit_client_data {
+
+class ModuleData;
+class ProcessData;
+
+// This class is used on the client to represent a function, containing a name, module path and
+// build ID, address and size.
+class FunctionInfo {
+ public:
+  FunctionInfo() = delete;
+  FunctionInfo(std::string pretty_name, std::string module_path, std::string module_build_id,
+               uint64_t address, uint64_t size)
```

Comment:
Nit; The ordering of the parameters feels a bit wrong. I would suggest the have the parameter for the module first and then the three parameters for the symbol.